README.md

```diff
@@ -1 +1,28 @@
-# lab-05
\ No newline at end of file
+# LAB 06
+
+## pratice of input and note test
+
+### Author: Blandine Dasilveira
+
+### Links and Resources
+
+- [submission PR](https://github.com/blandine-401javascript/lab-06/pull/1)
+
+- [swagger link](https://inspector.swagger.io/builder)
+
+- [get path](http://localhost:3000/products)
+
+- [post path](http://localhost:3000/products)
+
+
+![alt text](./asset/lab06.PNG)
+
+
+### Setup
+

```

Comment:
This section should detail how to run it, like start running the server in the command line.(with the command line).